app/(home)/plans/[branche]/page.tsx

```diff
@@ -29,18 +29,18 @@ export default async function Page(props: PageProps) {
   }
 
   branche.plans = await getPlans(branche);
-  if (!branche.plans.length) {
+  if (branche.plans.length) {
     return (
-      <div className="flex flex-col items-center justify-center font-Bree text-black h-[calc(100vh-416px)]">
+      <div className="flex flex-col items-center justify-center font-Bree text-black h-[calc(100vh-416px)] md:px-8 px-4 relative">
```

Comment:
```suggestion
      <div className="flex flex-col items-center justify-center font-Bree text-black h-[calc(100vh-416px)] md:px-8 px-4">
```

Tira relativo, pois o absolute foi removido.